src/utilities/vizSupport.py

```diff
@@ -1048,6 +1048,25 @@ def createCameraConfigMsg(viz, **kwargs):
     else:
         cameraConfigMsgPayload.renderMode = 0
 
+    if 'depthMapClippingPlanes' in kwargs:
+        if cameraConfigMsgPayload.renderMode != 1:
+            print('WARNING: vizSupport: depthMapClippingPlanes only works with renderMode set to 1 (depthMap).')
```

Comment:
:large_blue_circle:  We should discuss (maybe in subsequent PRs) moving away from using print statements for warnings etc. They do an 'end-run' around pythons exception and logging systems, whereas packages like `warnings` and `logging` allow us to capture and take particular action for out put that we provide for warnings/error/logs/exceptions.

Reply:
Agreed, would be nice to gradually update all these support files in the future to use these methods to expressing warnings.